firmware/hw_layer/mass_storage/create_ini_image.sh

```diff
@@ -14,7 +14,7 @@ BOARD_SPECIFIC_URL=$5
 
 echo "create_ini_image: ini $FULL_INI to $H_OUTPUT size $FS_SIZE for $SHORT_BOARDNAME [$BOARD_SPECIFIC_URL]"
 
-rm -f rusefi.zip ramdisk_image.h
+rm -f rusefi.ini.zip ramdisk_image.h
```

Comment:
root cause of the issue is copy-paste of magic file name, right?

while you are fixing the typo shall we go further and introduce a variable for file name and use the variable instead of magic file name twice?

Reply:
Ok done.

FYI my root cause was the HPFP change.  Apparently we're just tripping over the 112kB limit for the FAT filesystem even though the ZIP file is only 88kB.  I'm not sure where the extra overhead is coming from, but I reduced the size of the root to work around this for now.  Can we bump the 112kB for the image size?  If not we'll probably run into this again.